Automattic/wp-calypso
# Focused Launch: highlight selected plan in focused launch - detailed plans

### Changes proposed in this Pull Request

* In short, fixes #47814.

### Testing instructions

**Setup:**
1. Checkout this branch
2. Run `yarn start` in Claypso
3. Run `yarn dev --sync` in `apps/editing-toolkit`
4. Sandbox an unlaunched site on _a Free plan_ created via `/start`

**Test changes:**
1. Go to `/page/UNLAUNCHED_SITE_ID_CREATED_WITH_START.wordpress.com/home?flags=create/focused-launch-flow`
2. Click on "Launch" to open the Focused Launch modal
3. Select a plan from the Summary view (e.g. Premium plan)
4. Click on "View all plans"
5. Check that the selected plans is correctly highlighted in the plans grid as per #47814

Fixes #47814
